RoslynPluginGenerator/CommandLine/ProcessedArgs.cs

```diff
@@ -13,8 +13,9 @@ public class ProcessedArgs
         private readonly NuGetReference analyzerRef;
         private readonly string sqaleFilePath;
         private readonly string language;
+        private readonly bool acceptLicenses;
 
-        public ProcessedArgs(NuGetReference analyzerRef, string sqaleFilePath, string language)
+        public ProcessedArgs(NuGetReference analyzerRef, string sqaleFilePath, string language, bool acceptLicenses)
```

Comment:
Is there a better way to handle arguments that doesn't involve an ever-increasing number of parameters? I can see this getting out of hand as we add options for richer authoring experiences.